tests/phpunit/Mutator/Unwrap/UnwrapArrayMergeTest.php

```diff
@@ -285,6 +285,15 @@ function array_merge($array, $array1, $array2)
 $a = 'array_merge';
 
 $b = $a([1,2,3], [3,4,5]);
+PHP
+            ,
+        ];
+
+        yield 'It does not mutate when the input array is unpacked' => [
```

Comment:
Could you please also add a test for the following case:

```php
$first = [1, 2, 3, 4, 5];
$other = [[6, 7], [8]];

$result = array_merge($first, ...$other);
```

where only first (`0` index) item should be unwrapped

Reply:
Turns out my original solution didn't allow those, so i changed it to make that decision when mutating instead.